apps/webstore/src/app/shared/components/products/family-tree/family-tree-design/family-tree-design.component.ts

```diff
@@ -70,7 +72,7 @@ export class FamilyTreeDesignComponent implements AfterViewInit, OnInit, OnChang
   // FPS of the render loop
   framesPerSecond = 60;
   // design autosave frequency, in seconds
-  autosaveFrequencyInSeconds = 30;
+  autosaveFrequencyInSeconds = 15;
```

Comment:
Why have you increased the autosave frequency? I don't see it mentioned in the issue, and increasing it will increase the load on the client's computer

Reply:
This was something Teo asked me to do while we were solving something else.